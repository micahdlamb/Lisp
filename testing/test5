(CAR (QUOTE ((QUOTE A) B C)))
(CAR (QUOTE (((QUOTE A)) B C)))
(CDR (QUOTE (((QUOTE A)) B C)))
(CDR (CONS (QUOTE NIL) (CONS (PLUS 3 4) (CONS (quote T) NIL))))
(CONS (QUOTE A) (QUOTE B))
(CONS (QUOTE (A . B)) (QUOTE B))
