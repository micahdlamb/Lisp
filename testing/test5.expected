(QUOTE A)
((QUOTE A))
(B C)
(7 T)
(A . B)
((A . B) . B)
